### PR TITLE
Add standard bundle

### PR DESCRIPTION
**Pull request by lfortunier**
- Use namespace after use core bundle in `PasswordSafeableTrait.php`
- Use bundle `smartbooster/standard-bundle`, changes in :
  - /bin
  - Add configs
  - Deletes make files
  - Add Kernel
  - Add .gitignore content
  - Changes require dev
  - Changes config of phpcs, phpstan and phpunit
  - Comment base test because it doesn't work with the new `smartbooster/standard-bundle` implementation
- Fix some style because of the new `smartbooster/standard-bundle` implementation

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -38,17 +38,17 @@ jobs:
           restore-keys: ${{ runner.os }}-composer-
```
Comment (mathieu-ducrot) [inline .github/workflows/ci.yml:38]: @lfortunier i think the composer cache step is messing with the flex recipe install that should add the make/qualimetry.mk and make/test.mk.

Try to remove the steps "Get composer cache directory" and "Cache composer dependencies"

File: bin/phpunit [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env php
```
Comment (mathieu-ducrot) [inline bin/phpunit:1]: I'm not sure this is required as our tests are run from vendor/bin/simple-phpunit

File: src/Loader/ParameterLoader.php [hunk 3]
```diff
@@ -11,7 +11,7 @@
  */
 class ParameterLoader
 {
-    const BATCH_SIZE = 20;
+    private const BATCH_SIZE = 20;
```
Comment (mathieu-ducrot) [inline src/Loader/ParameterLoader.php:14]: Are you sure that this shouldn't be public instead ?

Comment (lfortunier) [reply src/Loader/ParameterLoader.php:14]: I think no, this constant purpose is to be used in this class and not be used by others class.

File: make/install.mk [hunk 4]
```diff
@@ -0,0 +1,8 @@
+##
+## Installation and update
+## -------
+.PHONY: install
+install: ## Install the project
```
Comment (mathieu-ducrot) [inline make/install.mk:5]: Did you also remove the one from the global Makefile ?

**Comment by lfortunier** [comment]
I comment base test because there is an initialization error on database and Kernel. It could be work, but I don't solve it now.

**Pull request merged**